packages/vite/src/node/plugins/worker.ts

```diff
@@ -72,9 +97,13 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
           })}__`
         }
       }
-
+      let WorkerOptions = buildWorkerOptions(query)
```

Comment:
**nitpick (non-blocking):** `workerOptions` gets not reassigned and should use `camelCase`
```suggestion
      const workerOptions = buildWorkerOptions(query)
```